### ISSUE TITLE
Meetings view has a demo card for new folks

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/parabol#4635]
Blocked by #4624 

### Acceptance criteria
- Close this issue and do #4637 instead
- [New folks have a demo card on the Meetings view](https://www.figma.com/file/dfTQxhoKzohkCpxlztBbhS/Meetings-view?node-id=98%3A0) regardless of having completed the demo before signing up
- The demo card
  - Looks like a meeting card
  - Shows the connected viewer’s avatar and two active animal [scripted] avatars
  - They tap and go to the demo and the animal avatars match
  - They complete the demo by pressing ‘End Meeting’ and the demo card goes away
  - The more menu on the demo card:
    - Has disabled options for copying the invite link, copying the meeting URL, or adding to calendar so that folks learn where those options are if they pay attention here
    - Has an enabled option for removing the demo card from the Meetings view
    - They can remove it without having to complete it
    
### Estimated effort
- 20 hours

**Comment by ackernaut** [comment]
Closing in favor of #4637
